src/fr-webauthn/fr-webauthn.test.ts

```diff
@@ -0,0 +1,96 @@
+import { WebAuthnStepType } from './enums';
```

Comment:
Should we include copyright header on the test files for JS SDK as well?

Reply:
Ah, good catch! Will do that now.